stdlib/lazy.mli

```diff
@@ -63,15 +63,15 @@ external force : 'a t -> 'a = "%lazy_force"
    If [x] has already been forced, [Lazy.force x] returns the
    same value again without recomputing it.  If it raised an exception,
    the same exception is raised again.
-   Raise {!Undefined} if the forcing of [x] tries to force [x] itself
+   @raise Undefined if the forcing of [x] tries to force [x] itself
    recursively.
 *)
 
 val force_val : 'a t -> 'a
 (** [force_val x] forces the suspension [x] and returns its
     result.  If [x] has already been forced, [force_val x]
     returns the same value again without recomputing it.
-    Raise {!Undefined} if the forcing of [x] tries to force [x] itself
+    @raise Undefined if the forcing of [x] tries to force [x] itself
     recursively.
     If the computation of [x] raises an exception, it is unspecified
     whether [force_val x] raises the same exception or {!Undefined}.
```

Comment:
```suggestion

    If the computation of [x] raises an exception, it is unspecified
    whether [force_val x] raises the same exception or {!Undefined}.
        @raise Undefined if the forcing of [x] tries to force [x] itself
    recursively.
```

Reply:
can you remove your space before `@raise` so I can commiut from GitHub UI (and you get credited)